clang-tools-extra/clang-tidy/cppcoreguidelines/PreferAtOverSubscriptOperatorCheck.h

```diff
@@ -0,0 +1,40 @@
+//===--- PreferAtOverSubscriptOperatorCheck.h - clang-tidy ------*- C++ -*-===//
+//===--- PreferMemberInitializerCheck.h - clang-tidy ------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_CLANG_TOOLS_EXTRA_CLANG_TIDY_CPPCOREGUIDELINES_PREFERATOVERSUBSCRIPTOPERATORCHECK_H
+#define LLVM_CLANG_TOOLS_EXTRA_CLANG_TIDY_CPPCOREGUIDELINES_PREFERATOVERSUBSCRIPTOPERATORCHECK_H
+
+#include "../ClangTidyCheck.h"
+
+namespace clang::tidy::cppcoreguidelines {
+
+/// Enforce CPP core guidelines SL.con.3
```

Comment:
Maybe we can just replace the Doxygen comment above with what's in the  release notes + check description right now:

> Flags the unsafe ``operator[]`` and suggests replacing it with ``at()``.

What do you think?

Reply:
> Maybe we can just replace the Doxygen comment above with what's in the release notes + check description right now:
> > Flags the unsafe `operator[]` and suggests replacing it with `at()`.
> 
> What do you think?

@5chmidti just giving this a polite bump in case it got lost 🙂

Reply:
See the top-level review comment, but these three should always be the same.